### DISCUSSION TITLE
5  Timeseries – Modern Polars

### DISCUSSION THREAD

**Discussion by giscus** [kevinheavey/modern-polars#35] [General]
# 5  Timeseries – Modern Polars

A side-by-side comparison of the Polars and Pandas libraries.

https://kevinheavey.github.io/modern-polars/timeseries.html

**Comment by rodrigogiraoserrao** [comment]
Just a heads up that “5.5.2 Holiday calendars” is now outdated and Polars also has (some?) support for holiday calendars. :)

**Comment by williambdean** [comment]
Hey there. `group_by_dynamic` links seems to be stale
